models/Calendar.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const CalendarEventSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+  startTime: {
+    type: Date,
+    required: true,
+  },
+  endTime: {
+    type: Date,
+    required: true,
+  },
+  kidID: {
+    type: Schema.Types.ObjectId,
+    ref: 'Kid',
+    required: true,
+  },
+  createdBy: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  sharedWith: {
```

Comment:
All good to me but I am thinking should it be as an array to allow user sharing event with multiple users?